data/mt/retired/Jeremy-Trebas-b51a470c-840d-404c-aebe-a7cb47c3edff.yml

```diff
@@ -13,16 +13,19 @@ roles:
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '25'
 - start_date: 2021-01-04
+  end_date: '2022-12-31'
```

Comment:
@bfossen-ce this person was in the montana house from January 2021 through 01/02/2023, on 01/02/2023 they are starting in the senate